index.html

```diff
@@ -20,19 +26,13 @@
     </header>
 
     <section class="flex-container">
-      <h1 class="name">
-        I'm Milen <br />
-        Glad To See <br />
-        You!
-      </h1>
+      <h1 class="name">I'm Milen Glad To See You!</h1>
       <div>
         <p class="pharagraph">
-          I’m a software developer! I can help <br />
-          you build a product , feature or website Look through some of my work
-          and<br />
-          experience! If you like what you see and<br />
-          have a project you need coded, don’t<br />
-          hestiate to contact me.
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
         </p>
       </div>
 
```

Comment:
- [ ] Kindly make sure the **hading section** design sticks to the one from the Figma template like on elements positon, `height`, `spacing` between and around elements(top, right, bottom and left), `fonts`,...
### Figma design
![head_sect](https://user-images.githubusercontent.com/83042742/173114925-dd8893de-d4b8-4156-bab3-892e99c33684.png)
### Your design
![head_sec](https://user-images.githubusercontent.com/83042742/173114943-b4ce81f7-4efc-4849-b339-91a8c49fb412.png)